.gitignore

```diff
@@ -2,3 +2,6 @@
 tmp
 .bundle
 vendor
+Gemfile.lock
```

Comment:
Ok, though they are extremely annoying for contributors who want to make sure tests run on a clean checkout at the latest available gems. Yehuda Katz once wrote about checking them into Rails projects vs not checking them in Ruby libs.

Currently, it's locking the versions in development only, so it only affects you and contributors, not users. It's just extremely annoying if you add/update any development tool, use 'git add -p' and switch branches, rebase, etc.